.azure/pipelines/ci.yml

```diff
@@ -95,6 +95,7 @@ jobs:
             -pack
             -noBuildDeps
             $(_BuildArgs)
+      condition: ne(variables['Build.Reason'], 'PullRequest')
```

Comment:
Is it possible to enable this step when eng\SiteExtensions\** change? That would be a bit safer though this should be fine for now.

Reply:
Hmm not sure if that's possible, I'll ask some folks.

Reply:
So there's no first party support for this. Though now I think about it, it's still pretty fragile even if we implemented this ourselves. Often it's not changes we make that break it but instead underlying changes (in the SDK for example) that breaks this. I don't think this is a valuable improvement.

Reply:
:ok: let's get this in